README.rst

```diff
@@ -12,8 +12,8 @@ Enter the short name and primary class name of your new XBlock when prompted.
 
 To see your new XBlock in action, build and run a Docker image containing it::
 
-        $ docker build -t myxblock .
-        $ docker run -d -p 8000:8000 --name myxblock myxblock
+        $ docker build -t myxblock-dev $(pwd)
+        $ docker run -p 8000:8000 -v $(pwd):/usr/local/src/myxblock --name myxblock-dev myxblock-dev
```

Comment:
This line still didn't work for me when using the cookiecutter defaults. This did:

`docker run -p 8000:8000 -v $(pwd):/usr/local/src/myxblock-xblock --name myxblock-xblock-dev myxblock-xblock-dev`

And `make dev.run` did, which upon further consideration I think is probably better to suggest to users. It does image cleanup and is a lot easier to remember. :)